src/KendallQueue.java

```diff
@@ -0,0 +1,13 @@
+import lab1.KendallPacket;
+
+import java.util.ArrayDeque;
+import java.util.Queue;
+
+public class KendallQueue {
+
+    private Queue<KendallPacket> q;
+
+    public KendallQueue(int numPackets) {
+        this.q = new ArrayDeque<>(numPackets);
+    }
+}
```

Comment:
edit: this was a bit too complicated to do in one go. I tried putting a queue interface inside an interface and Java didn't like that lol.